### PR TITLE
Db Initialization

### PR DESCRIPTION
**Pull request by firuza0029**

### CODE REVIEW

File: InternshipBE/DAL/Database/ApplicationDbContext.cs [hunk 1]
```diff
@@ -6,10 +6,13 @@ namespace DAL.Database
 {
 	public class ApplicationDbContext : IdentityDbContext<User>
 	{
-		public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options) 
+        public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options) : base(options)
```
Comment (romanstat) [inline InternshipBE/DAL/Database/ApplicationDbContext.cs]: padding

File: InternshipBE/DAL/Migrations/20211026181547_SeedData.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+using Shared.Enums;
+using System;
+
+namespace DAL.Migrations
+{
+    public partial class SeedData : Migration
+    {       
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {           
+
```
Comment (romanstat) [inline InternshipBE/DAL/Migrations/20211026181547_SeedData.cs]: image

Comment (romanstat) [reply InternshipBE/DAL/Migrations/20211026181547_SeedData.cs]: image

Comment (romanstat) [reply InternshipBE/DAL/Migrations/20211026181547_SeedData.cs]: hash the password!

File: InternshipBE/DAL/Migrations/20211026181547_SeedData.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            migrationBuilder.InsertData("InternshipStacks", new[] { "Id", "InternshipId", "TechnologyStackType" }, new object[] { 2, 2, 5 });
+
+            migrationBuilder.InsertData("InterviewInvites", new[] { "Id", "UserCandidateId", "ContactDate" }, new object[] { 1, 1, DateTime.Now });
+
+            migrationBuilder.InsertData("InterviewInvites", new[] { "Id", "UserCandidateId", "ContactDate" }, new object[] { 2, 2, DateTime.Now });                               
+
+            migrationBuilder.InsertData("UserInternships", new[] { "Id", "UserId", "InternshipId" }, new object[] { 1, 1, 1 });
+
+            migrationBuilder.InsertData("UserInternships", new[] { "Id", "UserId", "InternshipId" }, new object[] { 2, 2, 2 });
```
Comment (romanstat) [inline InternshipBE/DAL/Migrations/20211026181547_SeedData.cs]: use enums, example - RoleType.Admin

**Review by romanstat — APPROVED** [review]

**Review by Nikityanov — APPROVED** [review]

**Review by Evgeniy-Sinicin — APPROVED** [review]

**Pull request merged**